docs/source/serving/sky-serve.rst

```diff
@@ -62,7 +62,7 @@ Here is a simple example of serving an LLM model (:code:`lmsys/vicuna-13b-v1.5`)
             # Fields below describe each replica.
             resources:
               ports: 8080
-              accelerators: A100:1
+              accelerators: {L4:8, A10g:8, A10:8, A100:4, A100:8, A100-80GB:2, A100-80GB:4, A100-80GB:8}
```

Comment:
This picks Azure's A100-80GB:2 for me and results in the same error
```

(task, pid=7600)   File "/home/azureuser/miniconda3/envs/vllm/lib/python3.9/site-packages/torch/distributed/distributed_c10d.py", line 2050, in all_reduce
(task, pid=7600)     work = group.allreduce([tensor], opts)
(task, pid=7600) torch.distributed.DistBackendError: NCCL error in: ../torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp:1333, unhandled system error (run with NCCL_DEBUG=INFO for details), NCCL version 2.18.1
(task, pid=7600) ncclSystemError: System call (e.g. socket, malloc) or external library call failed or device error.
(task, pid=7600) Last error:
(task, pid=7600) XML Import Channel : dev 2 not found.
```
Did you not encounter this error?

Is there a good way to ban Azure:
```

I 01-05 14:13:07 optimizer.py:910] ----------------------------------------------------------------------------------------------------------
I 01-05 14:13:07 optimizer.py:910]  CLOUD   INSTANCE                    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN
I 01-05 14:13:07 optimizer.py:910] ----------------------------------------------------------------------------------------------------------
I 01-05 14:13:07 optimizer.py:910]  Azure   Standard_NC48ads_A100_v4    48      440       A100-80GB:2    eastus          7.35          ✔
I 01-05 14:13:07 optimizer.py:910]  GCP     g2-standard-96              96      384       L4:8           us-east4-a      7.98
I 01-05 14:13:07 optimizer.py:910]  GCP     a2-ultragpu-2g              24      340       A100-80GB:2    us-central1-a   10.06
I 01-05 14:13:07 optimizer.py:910]  Azure   Standard_NC96ads_A100_v4    96      880       A100-80GB:4    eastus          14.69
I 01-05 14:13:07 optimizer.py:910]  GCP     a2-highgpu-4g               48      340       A100:4         us-central1-a   14.69
I 01-05 14:13:07 optimizer.py:910]  AWS     g5.48xlarge                 192     768       A10G:8         us-east-1       16.29
I 01-05 14:13:07 optimizer.py:910]  GCP     a2-ultragpu-4g              48      680       A100-80GB:4    us-central1-a   20.11
I 01-05 14:13:07 optimizer.py:910]  Azure   Standard_ND96asr_v4         96      900       A100:8         eastus          27.20
I 01-05 14:13:07 optimizer.py:910]  GCP     a2-highgpu-8g               96      680       A100:8         us-central1-a   29.39
I 01-05 14:13:07 optimizer.py:910]  AWS     p4d.24xlarge                96      1152      A100:8         us-east-1       32.77
I 01-05 14:13:07 optimizer.py:910]  Azure   Standard_ND96amsr_A100_v4   96      1924      A100-80GB:8    eastus          32.77
I 01-05 14:13:07 optimizer.py:910]  GCP     a2-ultragpu-8g              96      1360      A100-80GB:8    us-central1-a   40.22
I 01-05 14:13:07 optimizer.py:910]  AWS     p4de.24xlarge               96      1152      A100-80GB:8    us-east-1       40.97
I 01-05 14:13:07 optimizer.py:910] ----------------------------------------------------------------------------------------------------------
```

Reply:
I didn't encountered this error; My controller chooses two L4:8 replicas